### PR TITLE
[Term Entry] Go Math Functions Round()

### PR DESCRIPTION
**Pull request by 4hbab**
- Added math.Round() term for go
- Closes Issue #2560 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/go/concepts/math-functions/terms/round/round.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Math'
+  - 'Functions'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`Round()`** function in Go is used to round a given number to the nearest integer. This function is part of the `math` package and can be imported to use in your Go programs.
```
Comment (SSwiniarski) [inline content/go/concepts/math-functions/terms/round/round.md:15]:
```suggestion
The **`Round()`** function in Go is used to round a given number to the nearest integer. This function is part of the `math` package which needs to be imported to use in Go programs.
```

File: content/go/concepts/math-functions/terms/round/round.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+## Syntax
+
+```pseudo
+result := math.Round(x)
+```
+
+Where:
+
+- `x`: is the number to be rounded
```
Comment (SSwiniarski) [inline content/go/concepts/math-functions/terms/round/round.md:25]:
```suggestion
- `x`: is the number to be rounded.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@4hbab I did a first review and left a couple of comments.

**Comment by 4hbab** [comment]
@SSwiniarski I've made the changes

**Review by SSwiniarski — APPROVED** [review]
@4hbab looks good for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@4hbab @SSwiniarski Looks good to merge

**Comment by SSwiniarski** [comment]
@4hbab Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/go/math-functions/round

image

**Pull request merged**